early July - Easily fix errors on your connections (#660)
akeneo/pim-helpcenter: content/updates/2020-07/1-easily-fix-errors-on-your-connections.md

> # Easily fix errors on your connections 

Reviewer: Don't forget the yaml at the beginning of the file to generate the PIM updates in the comm panel. You can find an example of the format in master (see old updates). :)

Author: Yes, I created the PR before our Tuesday meeting :) I'll add it at the beginning of the page

> ::: meta-data type="Improvement" features="Connections, Productivity" available="early-July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"

Reviewer: ```suggestion
::: meta-data type="Improvement" features="Connections, Productivity" available="early July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
```

:::

> We improved the connection error messages by adding several information that should help you to fix them.

Reviewer: ```suggestion
We improved the connection error messages by adding information that will help you to fix them.
```

In the `Monitoring` page, you will now find links to other PIM page or help center pages, and some data about the related product.  

> ![Enriched error message](../img/fix-errors-on-your-connections.png)

Reviewer: The image is more than 793px wide. Also, text on the screenshot is cropped, is it on purpose?
